client/scripts/views/pages/view_proposal/index.ts

```diff
@@ -1298,14 +1298,10 @@ const ViewProposalPage: m.Component<
               (proposal as OffchainThread).snapshotProposal?.length > 0) &&
             m(ProposalSidebarLinkedViewer, {
               proposal,
-            }),
-          proposal instanceof OffchainThread &&
-            (isAuthor || isAdmin) &&
-            m(ProposalSidebarStageEditorModule, {
-              proposal,
               openStageEditor: () => {
                 vnode.state.stageEditorIsOpen = true;
               },
+              showAddProposalButton: (isAuthor || isAdmin)
```

Comment:
we should check that the proposal is an instance of OffchainThread here. related threads is only for OffchainThreads at the moment. 

`proposal instanceof OffchainThread && (isAuthor || isAdmin)` should do the trick

Reply:
That's getting checked on line 1296 though right? ProposalSidebarLinkedViewer won't be rendered otherwise, or am I missing something

Reply:
Oh yeah you're good